dist/common/scripts/scylla-housekeeping

```diff
@@ -181,7 +184,7 @@ if args.config != "":
     if not os.path.isfile(args.config):
         traceln("Config file ", args.config, " is missing, terminating")
         sys.exit(0)
-    config = configparser.SafeConfigParser()
+    config = configparser.ConfigParser()
```

Comment:
how is that relates to the change?

Reply:
I change this by a single patch.

when I see DeprecationWarning &  message `Unable to retrieve version information` , it seems something is wrong.